ios/RNKommunicateChat.swift

```diff
@@ -548,7 +550,7 @@ class RNKommunicateChat : RCTEventEmitter, KMPreChatFormViewControllerDelegate,
     }
     
     open override func addListener(_ eventName: String!) {
-       Kommunicate.subscribeCustomEvents(events: [CustomEvent.messageReceive, CustomEvent.messageSend,CustomEvent.faqClick, CustomEvent.newConversation, CustomEvent.submitRatingClick, CustomEvent.restartConversationClick, CustomEvent.richMessageClick, CustomEvent.conversationBackPress, CustomEvent.conversationListBackPress ], callback: self)
+       Kommunicate.subscribeCustomEvents(events: [CustomEvent.messageReceive, CustomEvent.messageSend,CustomEvent.faqClick, CustomEvent.newConversation, CustomEvent.submitRatingClick, CustomEvent.restartConversationClick, CustomEvent.richMessageClick, CustomEvent.conversationBackPress, CustomEvent.conversationListBackPress, CustomEvent.conversationInfoClick ], callback: self)
```

Comment:
@amntoppo  There will be one more method called `supportedEvents()`. Add the new event in that method as well.